### DISCUSSION TITLE
Subscribed to paid v0.dev plan but am unable to access

### DISCUSSION THREAD

**Discussion by thefarmer2000** [vercel/community#5010] [Account Issue]
### Question

As the title states, I purchased the plan and have been on the wait list for a while but am unable to access the generator.

Please advise.

Username on vercel is polarb3ar

**Comment by amyegan** [comment]
Thanks for letting us know! I'm checking with the v0 team about it now.

**Reply by thefarmer2000** [reply]
Thank you!

**Reply by thefarmer2000** [reply]
Any update on this?  Will I be provided with an e-mail invite so I can use the product?

**Reply by amyegan** [reply]
Please check your email and let me know if you received your invite 🙏

**Comment by thefarmer2000** [comment]
All set!  Thank you!!
________________________________
